zentral/core/probes/templates/probes/index.html

```diff
@@ -76,7 +76,7 @@ <h2 class="mb-3">Probe{{ paginator.count|pluralize }} ({{ paginator.count }})</h
     </div>
 
     {% pagination next_url previous_url %}
-{% else %}
+{% elif form.has_changed %}
```

Comment:
Doesn't work, because active is already set in the form. Also, the link doesn't really links to all the probes.

There is a `is_initial` method on the form. Or maybe because it is not that big of an issue, we just remove the active by default, and have simply the same mechanism as with the other ones.

Reply:
Thanks for your review.
There's a decision to be taken. Do we remove the initial state? or we keep it?

Reply:
Remove the default to active. Make it work like the other search forms please.